### PR TITLE
MINITS - ITS-like operating system for the PDP-11

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
This probably has the lowest priority of all tickets, but I'll include it for completeness.

MC: MINITS; MITS.B; MTIS.S;

Edit: Not so low priority any more.

### CODE REVIEW

File: src/mits_s/config.849 [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 .if z %opsys-"IT		;if under ITS
 	.macro $insrt fn1,fn2
-		.insrt MC:MITS.S;fn1 fn2
+		.insrt MITS.S;fn1 fn2
```
Comment (atsampson) [inline src/mits_s/config.849]: Veeery picky comment, but: do the version bump as part of this commit?

Comment (larsbrinkhoff) [reply src/mits_s/config.849]: Thanks, fixed.

**Comment by larsbrinkhoff** [comment]
Note that if you strip off the DZ11 and CH11, this runs on 11SIM.  The DL11 at address 177560 is compatible with the standard PDP-11 console.

**Comment by larsbrinkhoff** [comment]
I found this in the configuration file for MINITS: 

```
         75 <==> Atari Sunnyvale
         76 <==> Atari-MIT land line
         77 <==> Atari Cambridge Bridge
```

So I'm guessing there was a Chaosnet connection between MIT and Atari Cambridge Research.

@slburson, you wrote to Hacker News "*In 1983 I was hacking on Symbolics Lisp Machines at Atari Cambridge Research.*"  Is it true there was a Chaosnet with a link to MIT?

**Pull request merged**